todo_app/data/trello_items.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import json
+from todo_app.data.item import Item
+from todo_app.data.trelloList import TrelloList
+import todo_app.data.trello_constants as constants
+
+class Trello_service(object):
+    trello_lists = {}
+    items = []
+
+    def __init__(self):
+        self.get_trello_secrets()
+        self.get_lists()
+        self.get_items_from_trello()
+
+    def get_trello_secrets(self):
+        trello_key = ''
+        trello_token = ''
+        trello_secrets = []
+        with open(constants.TRELLO_KEYS_PATH, 'r') as file:
+            split_lines = [line.split('=') for line in file.read().splitlines()]
+            trello_secrets = [{'k': k, 'v': v} for [k, v] in split_lines]
+        
+        for item in trello_secrets:
+            k = item['k']
+            v = item['v']
+            if k == "key":
+                trello_key = v
+            else:
+                trello_token = v
+        
+        constants.TRELLO_CREDENTIALS = f"key={trello_key}&token={trello_token}"
```

Comment:
It would be simpler to add the key and token as environment variables. There is a .env file (gitignored but .env.template provides an example), which is automatically loaded when you run the app. 

So you can add new env vars to .env and then this function can be simplified to e.g.
```
trello_key = os.getenv('TRELLO_KEY')
trello_token = os.getenv('TRELLO_TOKEN')
constants.TRELLO_CREDENTIALS = f"key={trello_key}&token={trello_token}"
```

env vars are a bit more convenient, and will help with next module's exercise.

Also the board id, while not a secret, can still be set via an env var, so that the code can be run in different environments without any changes to the committed files. And again this will help with tests next module.